lohica-js-network/src/index.js

```diff
@@ -3,9 +3,22 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
+import { createStore } from 'redux';
+import allReducers from './reducers/rootReducer';
+import { Provider } from 'react-redux';
 
-ReactDOM.render(
-    <App />,
-    document.getElementById('root')
-);
+
+const store = createStore (allReducers);
+function render () {
+    ReactDOM.render(
+        <Provider store = {store}>
+            <App store = {store}/>
```

Comment:
You transfer store to the provider, so there is no need to transfer it to App component